Styling/README.md

```diff
@@ -1344,6 +1347,66 @@ For example:
+| `font` | `String` | The font of the displayed text. | `"30px sans-serif"` |
+| `backgroundColor` | `vec4` | The label background color. | `rgba(42, 42, 42, 0.8)` |
+| `backgroundPadding` | `vec2` | The background padding, in pixels, of this label.  The `x` value controls horizontal padding, and the `y` value controls vertical padding. The number of `x` pixels is added to both sides. The number of `y` pixels is added to the top and bottom. | `vec2(7, 5)` |
+| `backgroundEnabled` | `Boolean` | Whether or not to display the label background. | `false` |
+| `scaleByDistance` | `vec4` | Sets near and far scaling properties of a feature based on the features's distance from the camera. A feature's scale will interpolate between the `y` and `w` values while the camera distance falls within the lower, `x`, and upper, `z`, bounds. Outside of these ranges, the features's scale remains clamped to the nearest bound.  If undefined, `scaleByDistance` will be disabled. | `undefined` |
+| `translucencyByDistance` | `vec4` | Sets near and far translucency properties of a feature based on the feature's distance from the camera. Translucency describes how transparent the point will be, where `0.0` means the point is completely transparant and `1.0` means the point is completely opaque. A feature's translucency will interpolate between the `y` and `w` while the camera distance falls within the upper and lower bounds of the specified `x` and `z`. Outside of these ranges the feature's translucency remains clamped to the nearest bound.  If undefined, `translucencyByDistance` will be disabled. | `undefined` |
+| `distanceDisplayCondition` | `vec2` | Sets the condition specifying at what distance from the camera that this feature will be displayed. The `x` value is the smallest distance in the interval where the feature is visible. The `y` value is the largest distance in the interval where the object is visible. If undefined, `distanceDisplayCondition` will be disabled. | `undefined` |
+| `heightOffset` | `Number` | The distance, in meters, to offset the height from the original height of the point. This is ignored if the value is less than or equal to zero. | `0.0` |
+| `anchorLineEnabled` | `Boolean` | Whether or not to display a line from the feature to the point's original position. This is ignored unless `heightOffset` is greater than zero. | `false` |
+| `anchorLineColor` | `vec4` | The color of the line from the feature to point's original position. This is ignored unless `heightOffset` is greater than 0.0. | `color('#FFFFFF')` |
+| `image` | `String` | A URL to an image or a data URI to be displayed instead of a point. | `undefined` |
+| `disableDepthTestDistance` | `Number` | The distance from the camera at which to disable the depth test to, for example, prevent clipping against terrain. When set to zero or `undefined`, the depth test is always applied. When set to Number.POSITIVE_INFINITY, the depth test is never applied. | `undefined` |
```

Comment:
Include `in meters`.

Reply:
I haven't tested this, but `Number.POSITIVE_INFINITY` probably won't get parsed correctly by the styling language. We'll need to include this number in the [constants section](https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/596d30eb327203f87f443615626ba2acb2a70e0e/Styling/README.md#constants)  and also added to Cesium.